### PR TITLE
Template updates oct23

### PR DESCRIPTION
**Pull request by chfoster**
## Pull request overview

Updates for accessibility, UI, CSS and cookie consent based on updates made to the [template](https://github.com/dfe-analytical-services/shiny-template) in the past year. 

## Pull request checklist

Please check if your PR fulfils the following:
- [x] Tests have been run locally and are passing (`run_tests_locally()`)
- [x] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

## What is the new behaviour?

- Cookie consent added n new gov styling 
- User testing removed from beta banner
- Updates to UI in line with latest template
- Updates to CSS
- Year removed from EES hyperlink so link always goes to latest data

Tests need adding - this will be done in a separate branch.

## Anything else

The CSS and UI updates have changed the appearance slightly. Please check you're happy with the news styling. 

@rmbielby - I noticed in[ KS4 ready reckoners](https://github.com/dfe-analytical-services/KS4-ready-reckoners) you updated the tables to Reactables - is this recommended here as well? I haven't edited the tables or charts but let me know if you have suggestions there.

### CODE REVIEW

**Comment by rmbielby** [comment]
On tables, I'm generally picking up the sense that reactable offers us more flexibility and some cleaner styling by default. I changed the KS4RRs to reactable partially just to get some more experience of using reactable myself as I'd not actuallt used iut so much. But I did find it a bit neater than datatables.

For charts, I'm going on the assumption that both of grey background and gridlines are non-ideal in terms of accessibility, hence switching everything to theme_classic() on KS4RRs. I think this makes sense to apply as general guidance across all dashboards. Again, I think this looks cleaner in the context of the dashboards than ggplot defaults anyway as well as improving the accessibility a little.

File: www/dfe_shiny_gov_style.css [hunk 1]
```diff
@@ -56,6 +51,9 @@ a:active {
   text-decoration: underline
 }
```
Comment (rmbielby) [inline www/dfe_shiny_gov_style.css:53]: I've added some extra styling in the KS4RRS to put some padding in the cookie banner:
https://github.com/dfe-analytical-services/KS4-ready-reckoners/commit/54bf8e935b8dd601d7a5ea39ff1c5924b6499101

File: .hooks/pre-commit.R [hunk 2]
```diff
@@ -39,6 +39,11 @@ for (file in current_files$files) {
 if (error_flag) {
   cat("Warning, aborting commit. Unrecognised data files found, please update .gitignore or datafiles_log.csv.\n")
   quit(save = "no", status = 1, runLast = FALSE)
+} else {
+  tidy_output <- tidy_code()
```
Comment (rmbielby) [inline .hooks/pre-commit.R]: I've adjusted this in KS4RRs to be a little more sensible in how it works as discussed:
https://github.com/dfe-analytical-services/KS4-ready-reckoners/commit/54bf8e935b8dd601d7a5ea39ff1c5924b6499101

File: global.R [hunk 3]
```diff
@@ -9,6 +9,7 @@
 # renv::status()
 # Library calls ---------------------------------------------------------------------------------
 shhh <- suppressPackageStartupMessages # It's a library, so shhh!
+shhh(library(git2r, pos = 10000))
```
Comment (rmbielby) [inline global.R]: Can remove git2r now, as it's not needed after this morning's update to the commit hooks.

File: server.R [hunk 4]
```diff
@@ -18,6 +18,103 @@
...
+      } else {
+        msg <- list(
+          name = "dfe_analytics",
+          value = input$cookies$dfe_analytics
+        )
+        session$sendCustomMessage("analytics-consent", msg)
+        if ("cookies" %in% names(input)) {
+          if ("dfe_analytics" %in% names(input$cookies)) {
+            if (input$cookies$dfe_analytics == "denied") {
+              ga_msg <- list(name = paste0("_ga_", google_analytics_key))
```
Comment (rmbielby) [inline server.R:43]: Looks like the google_analytics_key hasn't been set. Needs to be defined in global.R and should be the digits from the GA tag excluding the "G-" from the start.

File: ui.R [hunk 5]
```diff
@@ -96,18 +94,12 @@ ui <- function(input, output, session) {
       logo_width = 150,
       logo_height = 32
     ),
+    shinyGovstyle::cookieBanner("DfE Key Stage 4 Transition Matrices"),
```
Comment (rmbielby) [inline ui.R]: Let's budge this up above the header like Cam suggested.

**Review by rmbielby — CHANGES_REQUESTED** [review]
Handful of minor changes. Reassuring how quickly you could put all these updates into the dashboard, in terms of getting them rolled out to other dashboards!

File: server.R (hunk 4 above)
Comment (chfoster) [reply server.R:43]: Just pushed https://github.com/dfe-analytical-services/Transition-matrices-dashboard/pull/12/commits/90779c5ffc439ed8c5fcafd265d7a5e7fb8c8e26  - is that what you meant?

Comment (rmbielby) [reply server.R:43]: Yep, looks good

**Review by rmbielby — APPROVED** [review]

**Pull request merged**